newsletter/backend/docs/newsletters.rst

```diff
@@ -0,0 +1,62 @@
+newsletters package
+===================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   newsletters.migrations
+   newsletters.tests
+
+Submodules
+----------
+
+newsletters.admin module
```

Comment:
Please refer to Know How sphinx article https://azeno.youtrack.cloud/articles/KH-A-3

Reply:
.rst files has been changed as instructed in a Know How article.